lib/engine/step/g_18_zoo/simple_draft.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require_relative '../base'
+
+module Engine
+  module Step
+    module G18ZOO
+      class SimpleDraft < Base
+        attr_reader :companies
+
+        ACTIONS_WITH_PASS = %w[bid pass].freeze
+
+        def setup
+          @companies = @game.companies.select { |company| company.phase == 'ISR' }.sort
+          @finished = false
+        end
+
+        def available
+          @companies
+        end
+
+        def may_purchase?(_company)
+          true
+        end
+
+        def auctioning; end
+
+        def bids
+          {}
+        end
+
+        def visible?
+          true
+        end
+
+        def players_visible?
+          true
+        end
+
+        def name
+          'Draft'
+        end
+
+        def description
+          'Draft Powers'
+        end
+
+        def finished?
+          @finished
+        end
+
+        def actions(entity)
+          return [] if finished?
+
+          entity == current_entity ? ACTIONS_WITH_PASS : []
+        end
+
+        def process_bid(action)
```

Comment:
what does this do differently from other process bids?

Reply:
there is not a real bid; every player can buy a company (paying the company price), or skip.
I didn't see any other bids in this way, maybe I miss any?

Reply:
i see got it